android/libraries/rib-android-core/src/main/kotlin/com/uber/rib/core/CoreAppCompatActivity.kt

```diff
@@ -81,4 +85,16 @@ abstract class CoreAppCompatActivity : AppCompatActivity() {
     super.onActivityResult(requestCode, resultCode, data)
     activityDelegate?.onActivityResult(this, requestCode, resultCode, data)
   }
+
+  /**
+   * [RibActivity] must call this since it does not use [ComponentActivity.setContentView] which
+   * already handles this.
+   */
+  private fun initViewTreeOwners() {
+    // Set the view tree owners before setting the content view so that the inflation process
+    // and attach listeners will see them already present
+    ViewTreeLifecycleOwner.set(window.decorView, this)
+    ViewTreeViewModelStoreOwner.set(window.decorView, this)
+    ViewTreeSavedStateRegistryOwner.set(window.decorView, this)
```

Comment:
I guess there are now `setViewTreeXyz` extensions that don't need to take in `this`

Reply:
this artifact was already there & the ext fun weren't visible so i stuck w/ this